py/picca/delta_extraction/data_catalogues/desi_data.py

```diff
@@ -27,7 +28,7 @@
 defaults = update_default_options(defaults, {
     "delta lambda": 0.8,
     "delta log lambda": 3e-4,
-    "blinding": "corr_yshift",
+    "blinding": "none",
```

Comment:
is it dangerous to have "none" as default here?

Reply:
nevermind, I think this is fine

Reply:
we can add another one, but then which one? I added this one in order not to have to keep changing it. In any case, I'm considering removing this option and just figuring out which blinding to set directly

Reply:
Right, I think it might be better to not have "blinding" as an option, but rather as something that is set internally

Reply:
blinding option was removed